mobile/src/main/java/org/openhab/habdroid/core/OpenHABVoiceService.java

```diff
@@ -178,40 +80,33 @@ private String extractVoiceCommand(Intent data) {
     }
 
 
-    private void sendItemCommand(final String itemName, final String command) {
+    private void sendItemCommand(final String itemName, final String command,
+                                 final Connection conn, final int startId) {
         Log.d(TAG, "sendItemCommand(): itemName=" + itemName + ", command=" + command);
         try {
-            performHttpPost(itemName, command);
+            performHttpPost(itemName, command, conn, startId);
         } catch (RuntimeException e) {
```

Comment:
Unrelated to this PR, I know, but: what is supposed to throw an exception here? To me, it looks like the try-catch can be removed and the below method can be inlined.

Reply:
I don't think so, too. As we already rewrote a lot of this class, I'll put this change into here, too :P